Gems/LmbrCentral/Code/Tests/EditorBoxShapeComponentTests.cpp

```diff
@@ -118,20 +121,28 @@ namespace LmbrCentral
     using EditorBoxShapeComponentManipulatorFixture =
         UnitTest::IndirectCallManipulatorViewportInteractionFixtureMixin<EditorBoxShapeComponentFixture>;
 
-    TEST_F(EditorBoxShapeComponentManipulatorFixture, BoxShapeNonUniformScaleSymmetricalEditingManipulatorsScaleCorrectly)
+    void SetUpBoxShapeComponent(
+        AZ::Entity* entity,
```

Comment:
suggestion: Maybe just pass `EntityId` her if we're only calling `GetId()` on the instance anyway